ui/utilities/sizing/_index.scss

```diff
@@ -5,7 +5,7 @@
 /// @type map
 /// @private
 $_sizes: (
-  'xxx-small,': $size-xxx-small,
+  'xxx-small': $size-xxx-small,
```

Comment:
This is actually covered in another PR (#751), but I needed this fix in order to be able to iterate over the map keys correctly.